app/lib/screens/identity_verification_screen.dart

```diff
@@ -1265,6 +1266,8 @@ class _IdentityVerificationScreenState
                 ],
               ));
     } on NotEnoughBalance catch (_) {
+      final wallets = await getPkidWallets();
+      final wallet = wallets.firstWhere((w) => w.type == WalletType.NATIVE);
```

Comment:
`firstWhere` will crash if the list is empty, please use `where`

Reply:
fixed.